scripts/actions/mobskills/questionmarks_needles.lua

```diff
@@ -1,6 +1,8 @@
 -----------------------------------
 -- ??? Needles
 -- Description: Shoots multiple needles at enemies within range.
+-- 'The Amigo Sabotender's special ability 1000 Needles has been renamed ??? Needles.''
```

Comment:
They renamed it but it still actually does a static 1k? That seems odd. Would make more sense if it was still semi random but on a diff scale than Cuijatender

Reply:
well, I got bad news.
![2023 11 18_220637](https://github.com/LandSandBoat/server/assets/6871475/81381b3f-ba4a-403f-9157-4f14d473336a)

Reply:
Just did 9497 on my next use. kinda OP for 75, but I get it for post 75 where our ws already can do more than that

Reply:
looks random between 1k and 9999 so far

Reply:
pets tp seems to matter, and looks like it does top out at 10k